File: 其他/折线图/curveView/curveView/ViewController.m
//
//  ViewController.m
//  curveView
//
//  Created by mingyue on 16/6/1.
//  Copyright © 2016年 G. All rights reserved.
//

#import "ViewController.h"
#import "CurveView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    NSArray *rateArray = @[@"2.100",@"4.000",@"5.110",@"3.900",@"6.930",@"8.000",@"3.100"];
    NSArray *dateArray = @[@"05-01",@"05-02",@"05-03",@"05-04",@"05-05",@"05-06",@"05-07"];
    CurveView *curveView = [[CurveView alloc]initWithFrame:CGRectMake(0, 100, 320, 162) Date:dateArray Rate:rateArray];
    curveView.backgroundColor = [UIColor whiteColor];
    [self.view addSubview:curveView];
    
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: UIResponder/buttonImage/buttonImage/ViewController.swift
//
//  ViewController.swift
//  buttonImage
//
//  Created by mingyue on 15-7-19.
//  Copyright (c) 2015年 csii. All rights reserved.
//
/**
 测试响应链
 */

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let button = UIButton(frame: CGRectMake(100, 100, 100, 30))
        button.backgroundColor = UIColor.red
        button.setTitle("button", for: .normal)
        button.addTarget(self, action: #selector(btnAction), for: .touchUpInside)
        self.view.addSubview(button)
        
        let iv = UIImageView(frame: CGRect(origin: CGPointMake(100, 100), size: CGSizeMake(100, 30)))
        iv.backgroundColor = UIColor.green
//        self.view.addSubview(iv)
        
        let iv1 = UIImageView(frame: CGRectMake(0,0, 100, 30))
        iv1.backgroundColor = UIColor.purple
        iv1.alpha = 1.0
        button.addSubview(iv1)
    }

    @objc func btnAction(){
        print("btnAction")
    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

